RFC-010-Omni-Layer.md

```diff
@@ -0,0 +1,120 @@
+# The Omni Layer Asset
+
+- RFC-Number: 010
+- Status: Draft
+- Discussion-issue: -
+- Created on: 27 Mar. 2019
+
+## Table of Contents
+
+<!-- toc -->
+
+  * [Description](#description)
+  * [The Omni Layer Assets](#the-omni-layer-assets)
+    + [`quantity`](#quantity)
+    + [`property_id`](#property_id)
+  * [Registry extension](#registry-extension)
+- [Examples](#examples)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC defines how the Bitcoin-based Omni Layer Asset MUST be described in Asset type headers.
+Omni Layer refers specifically to the [Omni Layer Protocol](https://github.com/OmniLayer/spec).
+
+The Asset type header was introduced in [RFC002](./RFC-002-SWAP.md) to describe assets being exchanged in a COMIT SWAP protocol.
+
+This in combination with subsequent RFCs that define concrete execution steps for particular SWAP protocols enables users to negotiate and execute Omni Layer asset swaps.
+
+## The Omni Layer Assets
+
+The Omni Layer Protocol defines several _assets_, also commonly referred as _coins_ or _tokens_ (from now on referred as Omni _asset_).
+It also allows the definition of new assets by the mean of a Bitcoin transaction containing specific data.
+
+This asset MUST be used on the Bitcoin Ledger, as defined in [RFC-004](./RFC-004-SWAP-Bitcoin.md).
+Subsequent RFCs MAY introduce the support of other ledgers for Omni Layer Assets.
```

Comment:
Is this even possible? Are Omni assets a thing on ledgers other than bitcoin?

Reply:
If not, then the hypothetical sentence could be removed.

Reply:
See question from Lloyd above: there is an implementation for Litecoin and Tether once wanted to move to Litecoin. So theoretically this would be possible... not sure how likely it is though. 
https://github.com/OmniLayer/omnicore-lite